### PR TITLE
add crash_exitcode to Forkserver to accomodate AFL_CRASH_EXITCODE

### PR DESCRIPTION
**Pull request by R9295**

### CODE REVIEW

File: libafl/src/executors/forkserver.rs [hunk 1]
```diff
@@ -1157,7 +1170,12 @@ where
 
         if let Some(status) = self.forkserver.read_st_timed(&self.timeout)? {
             self.forkserver.set_status(status);
-            if libc::WIFSIGNALED(self.forkserver().status()) {
+            let exitcode_is_crash = if let Some(crash_exitcode) = self.crash_exitcode {
+                libc::WSTOPSIG(self.forkserver().status()) == crash_exitcode
```
Comment (tokatoka) [inline libafl/src/executors/forkserver.rs:1174]: wfisignaled doesn't work here?

Comment (R9295) [reply libafl/src/executors/forkserver.rs:1174]: ``WIFSIGNALED`` returns a bool so we can't  compare it to a given exit code

Comment (domenukk) [reply libafl/src/executors/forkserver.rs:1174]: https://github.com/AFLplusplus/AFLplusplus/blob/1d17210d9fb0eb37ba866a3697643a9e4f37acd5/src/afl-forkserver.c#L2045

AFL++ uses `WEXITSTATUS` (and it's `i8`/`u8`, not `i32`?)

Comment (domenukk) [reply libafl/src/executors/forkserver.rs:1174]: Should we keep this closer to AFL++? @vanhauser-thc for visibility

Comment (R9295) [reply libafl/src/executors/forkserver.rs:1174]: @domenukk yeah you're right. its i8.

Comment (tokatoka) [reply libafl/src/executors/forkserver.rs:1174]: i think it should be close to AFLpp

**Comment by R9295** [comment]
@domenukk / @tokatoka  updated to use i8 and EXITSTATUS

**Pull request merged**